plugins/rapid7_vulndb/help.md

```diff
@@ -1,6 +1,6 @@
 # Description
 
-Make searching the Rapid7 vulnerability and exploit data fast, easy and efficient with the InsightConnect plugin. Leverage this curated repository of vetted computer software exploits and exploitable vulnerabilities to ensure your security operations are always aware of the latest threats that could be used against your environment
+The Vulnerability & Exploit Database plugin allows you to search and compare potential threats with a curated repository of vetted computer software exploits and exploitable vulnerabilities vulnerabilities
```

Comment:
Wondering if we should use the previous one here 😄

Reply:
Reading it, the previous looks far better. I'll change it now!